test/functional/Valhalla/playlist.xml

```diff
@@ -31,6 +31,9 @@
 			<!-- Use -XX:-EnableArrayFlattening. testDefaultValueInTriangleArray asserts the fields of each element are not NULL, 
 					which is not true as a FlatteningThreshold is set. -->
 			<variation>-Xgcpolicy:gencon -XX:ValueTypeFlatteningThreshold=12 -XX:-EnableArrayFlattening</variation>
+			<variation>-Xnocompressedrefs -Xgcpolicy:optthruput -XX:ValueTypeFlatteningThreshold=99999 -XX:+EnableArrayFlattening</variation>
+			<variation>-Xnocompressedrefs -Xgcpolicy:gencon -XX:ValueTypeFlatteningThreshold=99999 -XX:+EnableArrayFlattening</variation>
+			<variation>-Xnocompressedrefs -Xgcpolicy:gencon</variation>
```

Comment:
Does `-Xnocompressedrefs` work with 32bit SDK? Should we use Mode110 (-Xjit -Xgcpolicy:gencon -Xnocompressedrefs) for Line36? 
Line 35 can be `<variation>Mode110 -XX:ValueTypeFlatteningThreshold=99999 -XX:+EnableArrayFlattening</variation>`
Line 34 can be `<variation>Mode101 -XX:ValueTypeFlatteningThreshold=99999 -XX:+EnableArrayFlattening</variation>`

Reply:
> Does -Xnocompressedrefs work with 32bit SDK?

Currently, Valhalla builds are running on linux_x86-64 only. No 32-bit Valhalla SDK at this moment yet.

>Should we use Mode110 (-Xjit -Xgcpolicy:gencon -Xnocompressedrefs) for Line36?
>Line 35 can be <variation>Mode110 -XX:ValueTypeFlatteningThreshold=99999 -XX:+EnableArrayFlattening</variation>
>Line 34 can be <variation>Mode101 -XX:ValueTypeFlatteningThreshold=99999 -XX:+EnableArrayFlattening</variation>

Yes we can use modes here. Do we have documentation of all the test modes somewhere in OpenJ9 ?
I slightly prefer to see the configurations (reference mode and GC policy) directly from the command line rather than hiding them inside modes, which requires a lookup.

Reply:
>Do we have documentation of all the test modes somewhere in OpenJ9 ?

https://github.com/adoptium/TKG/blob/master/resources/modes.xml

>I slightly prefer to see the configurations (reference mode and GC policy) directly from the command line rather than hiding them inside modes, which requires a lookup.

yes, I do prefer explicitly writing out the JVM options. The benefit of modes is to help us to skip unnecessary/invalid platforms. In this case, Valhalla only runs linux_x86-64 atm, so I agree that we do not need to use modes.